zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -213,6 +224,21 @@ private void flushAndReschedule() {
     scheduleDelayedFlush();
   }
 
+  private void scheduleImportersCompletedCheck() {
+    controller.scheduleCancellableTask(
+        Duration.ofSeconds(10), this::checkImportersCompletedAndReschedule);
+  }
+
+  private void checkImportersCompletedAndReschedule() {
+    try {
+      importersCompleted =
+          searchEngineClient.importersCompleted(partitionId, configuration.getIndex().getPrefix());
+    } catch (final Exception e) {
+      LOG.warn("Unexpected exception occurred checking importers completed, will retry later.", e);
+    }
+    scheduleImportersCompletedCheck();
```

Comment:
❌ shouldn't we stop scheduling if the result was `true` once? 
in other words: we should keep checking only if it hasn't yet completed. or is my understanding wrong?